src/main/java/com/reimbursements/requests/CreateAuthorityRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.reimbursements.requests;
+
+public class CreateAuthorityRequest {
+	private String authorityName;
+	private int userAuthID;
+	

```

Comment:
Missing a constructor with two parameters. It's a good way of creating an object, to pass those parameters. Eventually `Lombok`s `@Data`  annotation is a good option, if you understand and accept what `Lombok` does underneath, and what the risk.